src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  width: 7 * ($day-size + $gap) - $gap + 2 * $calendar-padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $calendar-padding;
+
+  &__day {
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid #000;
+    background-color: #eee;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition-duration: 0.5s;
```

Comment:
The 'transition-duration' property should be moved to a more appropriate place. According to the checklist, transition properties should be included in the base state of the element, not in the hover state. Move 'transition-duration' to the base state of '&__day'.